.github/workflows/ci.yml

```diff
@@ -46,6 +46,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
```

Comment:
This change is needed to make pipelines passing on forks. Without it cloning fails.